samples/subsys/display/lvgl/boards/xiao_ble_nrf52840.conf

```diff
@@ -0,0 +1 @@
+CONFIG_INPUT=y
```

Comment:
Why does this need to be set here, versus being enabled in the shield's `Kconfig.defconfig`?

Reply:
My reasoning was that if it's enabled in the shield's `Kconfig.defconfig`, and the shield is used only for display and not using the input then more files than needed would be compiled. I thought I'd try to avoid that, but I can move it over there if that's not an issue.

Reply:
That does make sense. Typically in the `Kconfig.defconfig` for shields I have enabled `CONFIG_INPUT` when `CONFIG_LVGL` is set. Of course you can use the input subsystem without LVGL, but I would imagine most users of LVGL do want touch support if their display offers it.

Example: https://github.com/zephyrproject-rtos/zephyr/blob/89493ee56b6cbd51b1117796b5dc2a2666bc22e8/boards/shields/g1120b0mipi/Kconfig.defconfig#L1

Reply:
Fair enough, I'll do that

Reply:
Well, I thought I'd do the changes quick and easy, but something must have happened in the 250+ commits that happened between last time I looked at this and tonight, because after rebasing it's not working anymore...
I'll try to look into it some more later this week

Reply:
Changes made, I also fixed a missing display-inversion property in the gc9a01 node causing wrong colors to be displayed.